template_parser.py

```diff
@@ -89,6 +111,15 @@ def read_template_types(template_type, container_cmd, template_dir, plugins_dir)
                 sections = config.sections()
             except Exception as e:
                 sections = []
+            running_containers = subprocess.check_output("docker ps | awk \"{print \$NF}\"", shell=True).split("\n")
+            t_sections = []
+            # remove sections that represent running containers
+            for section in sections:
+                for container in running_containers:
+                    if section in container and template_type in container:
+                        t_sections.append(section)
+                        print section
+            sections = [x for x in sections if x not in t_sections]
```

Comment:
This checks for sections in a namespace's template, removing sections for containers that are already running. Is this a redundant check?